index.html

```diff
@@ -36,7 +37,129 @@ <h1>I’m Alexandra.<br>Glad to see you!</h1>
+                                        <P class="work-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                                        <ul class="blue-list">
+                                            <li>HTML</li>
+                                            <li>Css</li>
+                                            <li>javascript</li>
+                                        </ul>
+                                        <button class="work-btn btn">see project</button>
+                                    </div>
+                                </div>
+            </section>
+            <!--the about myself section--> 
+            <div class="about">
+                <h2>About Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <p id="ltsconnct">Let's CONNEECT</p>
+                <ul class="img-icons" id="last-icons">
+                    <li><a href="#"><img src="img/github.jpg" alt="twitter icon"></a></li>
+                    <li><a href="#"><img src="img/Linkedin icon.jpg" alt="Linked in icon"><a></li>
+                    <li><a href="#"><img src="img/angellist.jpg" alt="medium icon"></a></li>
+                    <li><a href="#"><img src="img/twitter.jpg" alt="github icon"></a></li>
+                    <li><a href="#"><img src="img/medium.jpg" alt="angellist icon"></a></li>
+                </ul>
+                <button class="about-btn btn">Get my Resume</button>
+            </div>
+                <div class="about-languages">
+                        <img class="iconss" src="img2/angle-down.png" alt="">
+                        <P>Languages</P>
+                </div>
+                <div class="container">
+                    <div class="lang-icon">
+                        <div class="lang">
+                            <img id="im" src="img2/Ellipse 1.png" alt="javascript">
+                            <p>Javascript</p>
+                        </div>
+                        <div class="lang">
+                            <img src="img2/Ellipse 2.png" alt="html">
+                            <p>HTML</p>
+                        </div>
+                        <div class="lang">
+                            <img src="img2/Ellipse 3.png" alt="css">
+                            <p>Css</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="about-languages">
+                    <img class="iconss" src="img2/angle-right.png" alt="vector">
+                    <P>Frameworks</P>
+                </div>
+                <div class="about-languages">
+                    <img class="iconss" src="img2/angle-right.png" alt="">
+                    <P>Skills</P>
+                </div>
```

Comment:
You're doing great 👍🏽 but make sure to apply the style on each of the items here, you're very close, take a look 👇🏽 

## Your design

![Screenshot 2023-04-30 at 00 28 20](https://user-images.githubusercontent.com/45739963/235335500-ff90aeb2-aa3c-4b0e-8b89-56eb1f02f2e8.png)

## Figma

![Screenshot 2023-04-30 at 00 29 14](https://user-images.githubusercontent.com/45739963/235335522-66ed0a7b-85e6-4258-98b2-965f1d2093d9.png)

Pay attention to the slightly gray background of the list of languages. 👀